test/unit/tasks/cms/cms_test.rb

```diff
@@ -2,24 +2,56 @@
 
 module Tasks
   class CmsTest < ActiveSupport::TestCase
+    class CmsTestFixPaginationHref < CmsTest
+      def setup
+        @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
+                                 published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
 
-    def setup
-      @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
-        published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
+        @page.save(validate: false)
+      end
 
-      @page.save(validate: false)
+      def test_fix_pagination_href
+        assert_equal true,  @page.published.include?('{{ part.link }}')
+        assert_equal false, @page.published.include?('{{ part.url }}')
```

Comment:
```suggestion
        assert @page.published.include?('{{ part.link }}')
        assert_not @page.published.include?('{{ part.url }}')
```
Or maybe this works?
```suggestion
        assert_includes @page.published, '{{ part.link }}'
        assert_not_includes @page.published, '{{ part.url }}'
```

Reply:
Done